site/content/en/development/development-guide.md

```diff
@@ -0,0 +1,171 @@
+---
```

Comment:
there is a new header here:
```
---
title: "development-guide"
linkTitle: "development-guide"
type: docs
weight: 4
description: >
    Developer guidance for how to contribute for AzureDisk CSI Driver.
---
```

It's hard to maintain when original page is updated? how should we handle this? remove header or use tool? 
@nilo19

Reply:
WDYM by the "original page"?

Reply:
e.g. there are two files `development-guide.md` in different places, if there is no automatic tool to sync content between these two files, one page will be outdated.

Reply:
yeah it's hard to sync, you could use hyperlink in one of the duplicates and keep all of the content in one markdown file. e.g., https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/docs/azure-permissions.md.